src/mgr/MgrStandby.cc

```diff
@@ -216,8 +216,8 @@ void MgrStandby::send_beacon()
   dout(10) << "sending beacon as gid " << monc.get_global_id() << dendl;
 
   map<string,string> metadata;
-#warning fixme, report addrs instead
   metadata["addr"] = client_messenger->get_myaddr_legacy().ip_only_to_str();
+  metadata["addrs"] = stringify(client_messenger->get_myaddrs());
```

Comment:
i think the only consumer of "addr" in ceph is `SAIAgent._collect_data()` in `src/pybind/mgr/diskprediction_cloud/agent/metrics/sai_agent.py`.